src/plotting.py

```diff
@@ -52,10 +31,10 @@ def plot_countries_kpis(
     )
 
 
-def plot_gdp_exp(selected_continent="All", selected_countries=None):
-    plot_data = get_continent_data_filtered_year(2007, selected_continent).sort_values(
-        "gdpPercap", ascending=False
-    )
+def plot_gdp_exp(selected_continent, selected_countries):
+    plot_data = get_continent_data_filtered_year(
+        "2007", selected_continent
```

Comment:
same story here with the year arg. Probably best to take a look at the plotting module on origin/main